src/test/java/com/microsoft/lst_bench/common/LSTBenchmarkExecutorTest.java

```diff
@@ -138,6 +138,58 @@ void testExperimentTimelineTelemetry() throws Exception {
     }
   }
 
+  /**
```

Comment:
This is not related to your patch, but I realized that `LSTBenchmarkExecutorTest` executes for all tests in the CI, including those that are using the Spark deployment in the containers, which will slow down CI run. Could you add an annotation `@DisabledIfSystemProperty(named = "lst-bench.test.db", matches = ".*")` to the class, e.g., as it is done for `ParserTest.java`?